packages/@lwc/rollup-plugin/src/index.ts

```diff
@@ -107,15 +107,15 @@ function appendAliasSpecifierQueryParam(id: string, specifier: string): string {
     return `${filename}?${params.toString()}`;
 }
 
-function transformWarningToRollupWarning(
+function transformWarningToRollupLog(
     warning: CompilerDiagnostic,
     src: string,
     id: string
-): RollupWarning {
-    // For reference on RollupWarnings, a good example is:
+): RollupLog {
+    // For reference on RollupWarnings/RollupLogs, a good example is:
```

Comment:
If we're renaming all references to RollupWarnings -> RollupLogs, is there still a need to keep the term here?

Reply:
```suggestion
    // For reference on RollupLogs (f.k.a. RollupWarnings), a good example is:
```